fk/teamcreate.go

```diff
@@ -33,6 +33,10 @@ import (
 )
 
 func teamCreate() exitCode {
+	if code := ensureUserIsntInATeam("Couldn't create team"); code > 0 {
```

Comment:
everywhere else we use `exitCode != 0` to test it, why use `exitCode > 0` now?

are you *sure* you mean "is a positive number" rather than "isn't zero"? (I have no idea if negative return codes are possible, but it wouldn't surprise me)